src/Providers/ServiceProvider.php

```diff
@@ -33,17 +33,25 @@ class ServiceProvider extends LaravelServiceProvider
     protected $defer = false;
 
     /**
-     * Perform post-registration booting of services.
+     * Register services.
      *
      * @return void
-     * @throws LaravelRabbitMqException
      */
-    public function boot()
+    public function register()
```

Comment:
I'm declaring `register()` which is an abstract method in the base service provider in Laravel 5.2. Also, I deemed it safe to move the methods that bind stuff to the service container here.

`publishConfig` and `registerCommands` are still better located inside the `boot()` method according to Laravel docs on packages.